src/Suggestions.js

```diff
@@ -1,69 +0,0 @@
-import React from "react";
-import Header from "./Header";
-import App from "./App";
-import { useState, useEffect } from "react";
-
-const Suggestions = ({ mood, socialize, location }) => {
-    const [startOver, setStartOver] = useState(false);
-    const [resultHeading, setResultHeading] = useState("");
-    const [resultImg, setResultImg] = useState("");
-
-    function handleRetakeQuiz(e) {
-        e.preventDefault();
```

Comment:
why do you have this here?

Reply:
Completely forgot to delete that line. I know it shouldn't be there since I am not using onSubmit so there wont be any bubbling up of events. Have already deleted that line.